executor/prepared_test.go

```diff
@@ -146,16 +146,16 @@ func (s *testSuite9) TestPlanCacheClusterIndex(c *C) {
 
 	// For point get
 	tk.MustExec(`prepare stmt2 from "select * from t1 where t1.a = ? and t1.b = ?"`)
-	tk.MustExec("set @v1 = 1")
-	tk.MustExec("set @v2 = 1")
+	tk.MustExec("set @v1 = '1'")
+	tk.MustExec("set @v2 = '1'")
 	tk.MustQuery("execute stmt2 using @v1,@v2").Check(testkit.Rows("1 1 111"))
 	tk.MustQuery("select @@last_plan_from_cache").Check(testkit.Rows("0"))
-	tk.MustExec("set @v1 = 2")
-	tk.MustExec("set @v2 = 2")
+	tk.MustExec("set @v1 = '2'")
+	tk.MustExec("set @v2 = '2'")
 	tk.MustQuery("execute stmt2 using @v1,@v2").Check(testkit.Rows("2 2 222"))
 	tk.MustQuery("select @@last_plan_from_cache").Check(testkit.Rows("1"))
-	tk.MustExec("set @v1 = 3")
-	tk.MustExec("set @v2 = 3")
+	tk.MustExec("set @v1 = '3'")
+	tk.MustExec("set @v2 = '3'")
```

Comment:
A related bug: https://github.com/pingcap/tidb/issues/19002. When the variable type and column type mismatches, point-get with plan-cache enabled would return wrong result. It should be fixed in another PR, so I modify the test cases here first, since the test purpose here is not on the type mismatch problem.